zeppelin-web/pom.xml

```diff
@@ -42,24 +42,6 @@
   </properties>
 
   <dependencies>
-    <dependency>
-      <groupId>org.apache.zeppelin</groupId>
-      <artifactId>spark-scala-2.11</artifactId>
```

Comment:
Hi @Reamer 
I added artifacts `spark-scala-2.11` `spark-scala-2.12` `spark-scala-2.13` in dependencies. 
It is because  I found that  ` mvn clean install -DskipTests -DskipRat -am -pl zeppelin-web -Pscala-2.11 -Pspark-scala-2.11 -Pspark-2.4 -Phadoop2 -Pweb-e2e -B
` wouldn't install them if not add them.

Reply:
Therefore, the profiles that add the dependencies.
In principle, the build looks good now. Unfortunately, the tests fail. I do not know why.

Reply:
> Therefore, the profiles that add the dependencies. In principle, the build looks good now. Unfortunately, the tests fail. I do not know why.

Yes . The test `test-selenium-with-spark-module-for-spark-2-4`  very easily fail.  Sometimes I need to rerun it many times.